_scripts/instruction-widget/templates/install/debian.html

```diff
@@ -11,3 +11,10 @@
 <pre>
 $ sudo apt-get install {{package}} {{backports_flag}}
 </pre>
+
+<aside class="note">
+    <h4>Note</h4>
+    <p>
+    The Debian packaged version of Certbot installs a cron job automatically into /etc/cron.d/certbot. This cronjob runs a renewal twice a day (0000, 1200) with a small offset. Note that this won't renew certificates that aren't within the renewal window, but you may still want to replace this with your own renewal workflow.
```

Comment:
> This cronjob runs a renewal twice a day (0000, 1200) with a small offset.

I think "runs a renewal" is a strange way to phrase it. Maybe just "This cron job runs `certbot renew` twice a day..."? 

I'm also personally not a fan of the "(0000, 1200)" and think the offset is irrelevant for most people, but if you like it, feel free to leave those parts as is.

> Note that this won't renew certificates that aren't within the renewal window...

How about something like "This will only renew certificates as they approach expiration" or even "This will only renew certificates that are in the renewal window"? Personally, not a fan of the double negative, but again, feel free to leave this as is if you prefer.